cli/lsp/diagnostics.rs

```diff
@@ -211,16 +212,16 @@ impl DiagnosticsServer {
 
                   let mark =
                     performance.mark("update_diagnostics_ts", None::<()>);
-                  let diagnostics =
-                    generate_ts_diagnostics(snapshot.clone(), &ts_server)
-                      .await
-                      .map_err(|err| {
-                        error!(
-                          "Error generating TypeScript diagnostics: {}",
-                          err
-                        );
-                      })
-                      .unwrap_or_default();
+                  let diagnostics = generate_ts_diagnostics(
+                    snapshot.clone(),
+                    &config,
```

Comment:
I had intended to flip diagnostics to not be calculated at all upfront if the config was disabled and then I must have completely forgotten to do the second half of that 🤦‍♂️